DecipherNow/gm-fabric-dashboard
# 637 settings view tests

Resolves #

## Proposed Changes

  -Adding tests for the Setting view and underlying PollingSettings component

## How to Test

  -Navigate to src/components/Main/components/Settings/index.test.js, remove 'x' before describe.
  -Navigate to src/components/Main/components/Settings/index.js and comment out ConfirmationModal
  -Run npm test

If the confirmation modal is not commented out, you will see this error "Enzyme Internal Error: unknown node with tag 4" which is due to react-modal using ReactDom.createPortal which is not yet supported in the upgraded version of Enzyme. Here is the PR that fixes the issue but not yet released: https://github.com/airbnb/enzyme/pull/1345

Description of the issue: https://github.com/airbnb/enzyme/issues/1150
 
Discussed this approach with David and he concurs that we can 'uncomment' that test at some point in the future when Enzyme is released and encompasses the feature that we need.
